PurchasesTests/Purchasing/PurchaserInfoTests.swift

```diff
@@ -1,5 +1,5 @@
 //
-//  PurchaserInfoTests.swift
```

Comment:
looks like the actual file rename is missing :)

Reply:
LOL. Nice. 🤦‍♂️

Reply:
✅